discover/discover.go

```diff
@@ -42,14 +42,15 @@ var port int64 = 9089
 // ConvertPolicy converts the knoxautopolicies to KubeArmor and Cilium policies
 func ConvertPolicy(c *k8s.Client, o Options) error {
 	gRPC := ""
+	targetPodName := "discovery-engine"
```

Comment:
lets call this targetSvc
```suggestion
	targetSvc := "discovery-engine"
```

Reply:
Yeah , will change `PodName` to `Svc`